src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetSchedulingAndDescription/index.tsx

```diff
@@ -0,0 +1,264 @@
+    id: 'component.DashboardVideoLiveWidgetSchedulingAndDescription.updateVideoSuccess',
+  },
+  updateVideoFail: {
+    defaultMessage: 'Video update has failed !',
+    description: 'Message displayed when video update failed.',
+    id: 'component.DashboardVideoLiveWidgetSchedulingAndDescription.updateVideoFail',
+  },
+  updateDescriptionBlank: {
+    defaultMessage: "Description can't be blank !",
+    description:
+      'Message displayed when the user tried to enter a blank description.',
+    id: 'component.DashboardVideoLiveWidgetSchedulingAndDescription.updateDescriptionBlank',
+  },
+});
+
+const schedulingFieldsTheme: ThemeType = deepMerge(theme, {
+  global: {
+    font: {
+      family: 'Roboto-Regular',
+    },
+  },
+  box: {
+    extend: null,
+  },
+  button: {
+    default: {
+      background: { color: 'white' },
+      border: { color: 'brand', size: 'xsmall' },
+      color: 'brand',
+      padding: { vertical: 'xsmall', horizontal: 'small' },
+    },
+    primary: {
+      background: { color: 'brand' },
+      border: undefined,
+      color: 'white',
+    },
+    border: {
+      radius: '4px',
+    },
+    size: {
+      large: {
+        border: {
+          radius: '6px',
+        },
+        pad: {
+          horizontal: '3rem',
+          vertical: '1rem',
+        },
+      },
+    },
+    extend: null,
+  },
+  formField: {
+    label: {
+      size: '0.8rem',
+      margin: '0.5rem 1rem 0',
+      color: 'bg-grey',
+    },
+    border: {
+      position: 'outer',
+      side: 'all',
+      color: 'blue-active',
+      style: 'solid',
+    },
+    round: {
+      size: 'xsmall',
+    },
+  },
+  textInput: {
+    extend: 'padding: 0 1rem 0.8rem',
+  },
+  maskedInput: {
+    extend: 'padding: 0 1rem 0.8rem',
+  },
+  dateInput: {
+    icon: {
+      size: '18px',
+    },
+  },
+  text: {
+    extend: 'font-size: 0.8em',
+  },
+});
+
+const StyledText = styled(Text)`
+  font-family: 'Roboto-Regular';
+`;
+
+interface DashboardVideoLiveWidgetSchedulingAndDescriptionProps {
+  video: Video;
+}
+
+export const DashboardVideoLiveWidgetSchedulingAndDescription = ({
+  video,
+}: DashboardVideoLiveWidgetSchedulingAndDescriptionProps) => {
+  const intl = useIntl();
+  const [descriptionState, setDescriptionState] = useState(video.description);
+
+  const descriptionMutation = useUpdateVideo(video.id, {
+    onSuccess: () => {
+      toast.success(intl.formatMessage(messages.updateVideoSuccess));
+    },
+    onError: (err) => {
+      report(err);
+      toast.error(intl.formatMessage(messages.updateVideoFail));
+      setDescriptionState(video.description);
+    },
+  });
+
+  const timeoutId = useRef<Maybe<number>>();
+
+  const debounce = (
+    fn: (updatedVideoProperty: Partial<Video>) => void,
+    ms = 500,
+  ) => {
+    return (updatedVideoProperty: Partial<Video>) => {
+      window.clearTimeout(timeoutId.current);
+      timeoutId.current = window.setTimeout(() => fn(updatedVideoProperty), ms);
+    };
+  };
```

Comment:
Extract this function once for all. We will not redefine it everywhere we need it.

Reply:
done